### DISCUSSION TITLE
Unable to bind a Smart Knob (TuVa ERS-10TZBVK-AA)

### DISCUSSION THREAD

**Discussion by allanstevens** [Koenkk/zigbee2mqtt#18294] [Devices]
Would anyone have any advice on how to bind a Smart Knob (TuVa ERS-10TZBVK-AA) to a light bulb (eWeLight ZB-CL01)

I first attempted to bind the smart knob directly to the bulb and it does seem to bind, but does not work when i press the button, I also get the error 'Failed to disable reporting for 'lampLoftSarah/1' cluster 'genOnOff'. Looking at the help docs, it mentions that if you get the reporting error create a group instead, so I have then attempted to create a group with the bulb and then bound the knob to the group but still no luck.

I have never binded anything before, so maybe i'm doing something stupid.  Which clusters should I use? So far I have tried LevelCtrl and OnOff.

I'm using the latest version of zibee2mqtt Would anyone be able to give me some advice?

Thank you!

**Comment by deviantintegral** [comment]
I posted about binding this device at https://github.com/Koenkk/zigbee2mqtt/discussions/19136.

This particular device has some quirks, especially as compared to the Philips Hue switches I have set up for direct binding.

**Reply by grafalex82** [reply]
This device has incorrect (at least weird) implementation of groups. Instead of binding to a light it requires adding a light and the knob to the same group. The knob can work only in one group at a time.
